app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -241,6 +241,9 @@ class SettingsFragment : PreferenceFragmentCompat() {
 
         val directions: NavDirections? = when (preference.key) {
             resources.getString(R.string.pref_key_sign_in) -> {
+                if (isDefaultBrowserExperimentBranch() && !isFirefoxDefaultBrowser()) {
+                    requireContext().metrics.track(Event.SetDefaultBrowserSettingsScreenClicked)
+                }
                 SettingsFragmentDirections.actionSettingsFragmentToTurnOnSyncFragment()
```

Comment:
This looks like it gets triggered when the users click sign in card instead of the default browser card, could you help me to clarify?

Reply:
Should this be `pref_key_make_default_browser`?

Reply:
It's the same preference id, regardless of the layout it is using or the order in the preference list. I used the same id to make the fewest changes in the Koltin code, I preferred to handle all in XML so it can be easier removed later.

Reply:
Thanks for catching that!

Reply:
No problem :)